Dockerfile

```
@@ -1,14 +1,17 @@
-FROM golang:1.20
+FROM ubuntu:latest
 
-RUN apt-get update && apt-get install -y vim git
-RUN mkdir -p /usr/src/gosh
-WORKDIR /usr/src/gosh
+RUN apt-get update && apt-get install -y vim git && apt install sudo
 
-# pre-copy/cache go.mod for pre-downloading dependencies and only redownloading them in subsequent builds if they change
-COPY go.mod go.sum ./
-RUN go mod download && go mod verify
+RUN sudo apt-get install software-properties-common -y
+
+RUN sudo apt update &&\
+    sudo add-apt-repository ppa:longsleep/golang-backports &&\
+    sudo apt install golang-go -y
```

おそらくですがこの辺のやつはsudoを消せると思います！でupdateも上でやっているのでここは以下のような感じでいいと思います！
```suggestion
RUN add-apt-repository ppa:longsleep/golang-backports &&\
    apt-get install golang-go -y
```

---

[こちら](https://github.com/product-dev-group/gosh/pull/1#discussion_r1104817032)で削除しました！